### PR TITLE
feat: add wyg-zooma

### PR DESCRIPTION
**Pull request by ccomben**
## Description

This PR introduces a new blog post, "Who You Gno - On the Record with Antoine Breuil." This post was written by [@ccomben](https://github.com/ccomben) on January 11, 2024.

### CODE REVIEW

File: posts/2024-01-24_wyg-zooma/README.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+Why does zôÖma believe social coordination is so important, and why do we need new tools for it? “We’ve always had tendencies to organize ourselves and tools defining rules for living together, diplomatic protocols for discussing between social groups, or trading goods and services. But almost all the tools that previous generations put in place are outdated. Our entire generation has lost confidence in institutions to allow groups of humans to organize, coordinate, and meet their needs. Our dependence on third parties who do not have the same interests as citizens is immense.”
+
+zôÖma believes that web3 holds the key to unlocking the emergence of new societies through products that are “unstoppable, resilient, and meet a real need,” whether for small villages in the south of France, Africa, or Asia or neighborhoods in Brazil or Korea. “We must have access to the radical transparency of institutions, the privacy of individuals, censorship-resistant tools, and autonomous communication from all commercial enterprises. It is on this solid foundation that civilizations that are more just and equitable can be built.”
+
+## Making Web3 More Accessible 
+
+
```
Comment (leohhhn) [inline posts/2024-01-24_wyg-zooma/README.md:31]:
```suggestion
```

File: posts/2024-01-24_wyg-zooma/README.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+According to zôÖma, this means abstracting away the concepts that everyday people don’t need to be aware of, such as networks, dApps, and even blockchain, “and always switching from one decentralized application to another.” Unifying (not centralizing) separate tools, networks, and technologies within a single, simple interface, he believes, is the key to broader adoption. “It's a very complex challenge, in terms of security, design, etc., but it's what I'm passionate about today.” 
+
+When it comes to Gno.land, Teritori has already delivered essential DAO tooling and standards, a Moderation DAO module to facilitate social communication and a Justice DAO module for conflict resolution. The team is now focusing on an on-chain project management tool to allow organizations and individuals to manage projects and track tasks smoothly and transparently on-chain.
+
```
Comment (leohhhn) [inline posts/2024-01-24_wyg-zooma/README.md:37]:
```suggestion
```

File: posts/2024-01-24_wyg-zooma/README.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+zôÖma says that humanity is at a turning point, and we must build the necessary tools now to avoid finding ourselves in a real-life version of George Orwell’s 1984. “I aspire to participate modestly in a world that is fairer, more transparent, and where society doesn’t need a puppet in a suit to improve its living conditions or respond to local needs. Web3 is just a tool, and if it doesn't meet this real need, then for me, it will be a failure.”
+
+*Experiment with Teritori today and test its Social Feed, which now includes Twitter-like functionality for posts, Medium-style articles, Soundcloud-inspired music, and videos—all based on Gno and IPFS and totally decentralized. You can also check out Teritori’s GnoModerationModule, which allows you to moderate a social network in a decentralized way. A faucet is available on the home page at [app.teritori.com](https://app.teritori.com/feed?network=gno-teritori).*
+
+
```
Comment (leohhhn) [inline posts/2024-01-24_wyg-zooma/README.md:49]:
```suggestion
```

**Review by leohhhn — APPROVED** [review]
Looks good 💯 
I've just left a couple of suggestions to remove some unnecessary new lines :)

File: posts/2024-01-24_wyg-zooma/README.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+authors: [christina]
+---
+
+Antoine Breuil, ‘zôÖma,’ is the co-founder of [Teritori](https://app.teritori.com/), an active Gno.land contributor and grantee that’s building key modules and tooling for Gno. A firm believer in equal opportunities, free and decentralized access to information, and helping fellow humans, zôÖma is fascinated by human behavior and how we organize ourselves, holding an avant-garde social experiment five years ago with Teritori co-founder ‘Pop.’ "La Suite du Monde" drew people across France to a small village in the countryside to create a shared community and society—with farmland, accommodation, and tools for common use.
+
+The goal was to form an in-real-life DAO whose members shared common goals and interests—using blockchain technology with a token to pay for goods and services and vote on governance matters. While many people participated and were enthusiastic about creating a shared society, zôÖma admits the experiment was early: no one was interested in interacting with the tech. “90% of people, rural or not, found it too complicated,” he admits. “We were a bit naive, but it was fascinating nonetheless.”
```
Comment (waymobetta) [inline posts/2024-01-24_wyg-zooma/README.md:11]:
```suggestion
The goal was to form an in-real-life DAO whose members shared common goals and interests using blockchain technology with a token to pay for goods and services and vote on governance matters. While many people participated and were enthusiastic about creating a shared society, zôÖma admits the experiment was early- no one was interested in interacting with the tech. “90% of people, rural or not, found it too complicated,” he admits. “We were a bit naive, but it was fascinating nonetheless.”
```

File: posts/2024-01-24_wyg-zooma/README.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+Like most creative types, zôÖma is used to spinning many plates in the air, overseeing La Suite du Monde while working as a freelance designer, front-end dev, and Artistic Director for an independent French record label. “Before entering the world of engineers, I founded and managed a collective for 12 years, which brought together artists from all disciplines, hackers, designers, tinkerers, to build some interesting projects.” La Suite de Monde allowed him to explore his passion for finding new approaches to social coordination first-hand. “I explored very radical things,” he says, “like the notion of “accepted by default” where anyone could use the collective budget by expressing their desire to do so three times. I wouldn’t recommend this,” he laughs, “but the experiments were fascinating and still serve me today in my work.”
```
Comment (waymobetta) [inline posts/2024-01-24_wyg-zooma/README.md:17]: > "..the notion of “accepted by default” where anyone could use the collective budget by expressing their desire to do so three times."

I am not sure what this means without further context. Does this mean that after 3 times of expressing interest in something that someone's idea is accepted without debate?

File: posts/2024-01-24_wyg-zooma/README.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+
+One really interesting trait about zôÖma is how he harnesses the creative and analytical sides of himself with equal application. Most people are predominantly right-brained or left-brained, yet, zôÖma is ‘ambidextrous’ in this regard. He’s a designer who’s created large-scale artistic events, cultural tours of Paris, and an award-winning independent movie documenting French artist and Bitcoin advocate Pascal Boyart, [The Underground Sistine Chapel](http://www.the-chapel.art), (which you should definitely check out!). Yet he’s also passionate about engineering and the future of cooperatives. He’s detail-driven and ambitious, taking his team at Teritori from two to 18 (14 full-time teammates and four part-time).
```
Comment (waymobetta) [inline posts/2024-01-24_wyg-zooma/README.md:19]:
```suggestion
One really interesting trait about zôÖma is how he harnesses the creative and analytical sides of himself with equal application. Most people are predominantly right-brained or left-brained, yet, zôÖma is ‘ambidextrous’ in this regard. He’s a designer who’s created large-scale artistic events, cultural tours of Paris, and an award-winning independent movie documenting French artist and Bitcoin advocate Pascal Boyart, [The Underground Sistine Chapel](http://www.the-chapel.art), (which you should definitely check out!). Yet he’s also passionate about engineering and the future of cooperatives. He’s detail-driven and ambitious, taking his team at Teritori from 2 to 18 (14 full-time teammates and four part-time).
```

File: posts/2024-01-24_wyg-zooma/README.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+
+When it comes to Gno.land, Teritori has already delivered essential DAO tooling and standards, a Moderation DAO module to facilitate social communication and a Justice DAO module for conflict resolution. The team is now focusing on an on-chain project management tool to allow organizations and individuals to manage projects and track tasks smoothly and transparently on-chain.
+
+## A Fairer, More Transparent World
+
+In 2024, Teritori enters a new phase called "Chapter II", which involves unifying all its work into a mobile and desktop application that could “trigger superb demonstrations of the potential of DAOs.” He enthuses, “I dream that we will see the emergence of a village that uses Teritori as a tool for internal discussion and co-financing. Will this be real in 2024? Who knows? But that’s where I focus all my energy!”
```
Comment (waymobetta) [inline posts/2024-01-24_wyg-zooma/README.md:39]:
```suggestion
In 2024, Teritori enters a new phase called "Chapter II," which involves unifying all its work into a mobile and desktop application that could “trigger superb demonstrations of the potential of DAOs.” He enthuses, “I dream that we will see the emergence of a village that uses Teritori as a tool for internal discussion and co-financing. Will this be real in 2024? Who knows? But that’s where I focus all my energy!”
```

File: posts/2024-01-24_wyg-zooma/README.md [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+
+He believes the internet has been a great leveler, enabling anyone with a connection to educate themselves on any subject; yet, the opportunity isn’t open to all, and free and open access is constantly diminishing. “I am a child of the internet. I grew up with warez, p2p, and an internet which provided me with daily resources to learn freely, everything that interested me. In some countries, it is impossible to benefit from this opportunity, and with the centralization of the internet on different key players, mass surveillance, and the censorship of certain dictators, the Internet is losing its very essence, which makes it magic. Distributed protocols can reshuffle the cards and offer tools for the public good.”
```
Comment (waymobetta) [inline posts/2024-01-24_wyg-zooma/README.md:41]:
```suggestion
He believes the Internet has been a great leveler, enabling anyone with a connection to educate themselves on any subject; yet, the opportunity isn’t open to all, and free and open access is constantly diminishing. “I am a child of the internet. I grew up with warez, p2p, and an Internet which provided me with daily resources to learn freely, everything that interested me. In some countries, it is impossible to benefit from this opportunity, and with the centralization of the Internet on different key players, mass surveillance, and the censorship of certain dictators, the Internet is losing its very essence, which makes it magic. Distributed protocols can reshuffle the cards and offer tools for the public good.” 
```

We should be consistent with the way we capitalize/don't capitalize terms.

File: posts/2024-01-24_wyg-zooma/README.md (hunk 5 above)
Comment (ccomben) [reply posts/2024-01-24_wyg-zooma/README.md:17]: Yes

File: posts/2024-01-24_wyg-zooma/README.md (hunk 6 above)
Comment (ccomben) [reply posts/2024-01-24_wyg-zooma/README.md:19]: @waymobetta We follow the [AP guidelines](https://guides.18f.gov/content-guide/our-style/numbers-and-percentages/#:~:text=Generally%20speaking%2C%20we%20follow%20the,10th%20or%20greater%20with%20numerals) when it comes to writing out numbers. In body copy, we write out numbers one to nine in full.

File: posts/2024-01-24_wyg-zooma/README.md (hunk 8 above)
Comment (ccomben) [reply posts/2024-01-24_wyg-zooma/README.md:41]: Agreed. I prefer to have it all lower-case, then. No one writes Internet with a capital I anymore; it's like writing out the world wide web.

Comment (ccomben) [reply posts/2024-01-24_wyg-zooma/README.md:41]: Sure, agree with consistency, but I prefer to keep to lowercase "i". I will change it. Thanks!

**Pull request merged**